src/Rules/Functions/CallToNonExistentFunctionRule.php

```diff
@@ -41,7 +41,7 @@ public function processNode(Node $node, Scope $scope): array
 			}
 
 			return [
-				RuleErrorBuilder::message(sprintf('Function %s not found.', (string) $node->name))->identifier('function.notFound')->discoveringSymbolsTip()->build(),
+				RuleErrorBuilder::message(sprintf('Function %s not found.', (string) $node->name))->identifier('function.notFound')->discoveringSymbolsTip((string) $node->name)->build(),
```

Comment:
Would be nice to do `(string) $node->name)` only once.

Reply:
If I understand your comment below, you actually don't want the argument at all in these cases, since here the symbols are for functions, not classes. Right?